johnwayodi/store-manager
# #161078815 Add view for all Sale Records

#### What does this PR do?
Adds view where the Store Attendant can view all their records, it is also used by the Administrator to view records by all Store Attendants
#### Description of Task to be completed?
Have the following ui components working
An individual Sale record when clicked should open a view containing details of the sale record i.e items, date, total price of items
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, **_store-manager_** by default
3 `git checkout ft-sales-records-all-161078815`
4. open **_sale-records-all.html_** on browser
#### Any background context you want to provide?
This view is to be available to both the Store attendant and the Administrator
1. For the Store Attendant, the view will display only sale records for the Store Attendant
2. For the Administrator, the view will display all records from all store attendants
#### What are the relevant pivotal tracker stories?
#161078815
#### Screenshots (if appropriate)
![sale-records-all](https://user-images.githubusercontent.com/10683903/46792794-d7692180-cd4c-11e8-823d-823c299d1254.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
